static/app/components/sparklines/index.tsx

```diff
@@ -1,8 +0,0 @@
-import {Sparklines as BaseSparklines} from 'react-sparklines';
```

Comment:
suspicious. is this package no longer used then?

Reply:
yeah 🚀 

46b03c8f

`knip` also has a mode to detect unused dependencies, I just haven’t enabled it yet :)